dts/bindings/modem/telit,me910g1.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright(c) 2023 Jeff Welder(Ellenby Technologies, Inc.)
+# SPDX - License - Identifier : Apache - 2.0
```

Comment:
This probably wants to be `SPDX-License-Identifier: Apache-2.0`

Reply:
@jeffwelder-ellenbytech please fix this.